src/hotspot/share/oops/inlineKlass.hpp

```diff
@@ -184,10 +184,10 @@ class ValueKlass: public InstanceKlass {
   bool is_inline_klass_slow() const        { return true; }
 
   // Casting from Klass*
-  static ValueKlass* cast(Klass* k);
+  static InlineKlass* cast(Klass* k);
 
   // Use this to return the size of an instance in heap words
-  // Implementation is currently simple because all value types are allocated
+  // Implementation is currently simple because all inline types are allocated
```

Comment:
This comment might need an update, because now, not all inline types are allocated in the Java heap.
May be just specifying this size only applies to heap allocated standalone instances.